api/db/migrations/20231027084224_add-role-column-to-certification-center-invitations-table.js

```diff
@@ -0,0 +1,16 @@
+const TABLE_NAME = 'certification-center-invitations';
+const COLUMN_NAME = 'role';
+
+const up = function (knex) {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.enu(COLUMN_NAME, ['MEMBER', 'ADMIN']);
```

Comment:
Il me semble qu'il est plus clair d'utiliser le nom complet de la commande Knex : `enum`, car c'est le mot que les développeurs ont en tête et vont rechercher dans le code, plutôt que le raccourci `enu` cf. https://knexjs.org/guide/schema-builder#enum-enu (même si l'exemple à cette référence est donné avec le raccourci `enu` 😕

```suggestion
    table.enum(COLUMN_NAME, ['MEMBER', 'ADMIN']);
```

Reply:
Ça a été modifié directement dans le commit d'origine 28887577d6151fb9302c61426b2284fcfb9f6af8